src/main/resources/application.yaml

```diff
@@ -454,6 +454,10 @@ stitching:
     url: "${EM_CCD_ORCHESTRATOR_URL:http://localhost:8084}/api/stitch-ccd-bundles"
   enabled: ${STITCHING_API_ENABLED:true}
 
+case-flags:
```

Comment:
maybe follow the _ pattern already in place rather than - ?

Reply:
I changed it to the same as other places

- https://github.com/hmcts/civil-service/pull/5001/files#diff-b4c0421bcb3280a8e738550c830dba197057c515bcdd9579a52cbfe9adca2061R447
- https://github.com/hmcts/civil-service/pull/5001/files#diff-b4c0421bcb3280a8e738550c830dba197057c515bcdd9579a52cbfe9adca2061R470

Reply:
but I do see other properties using _ and some using camelcase